conda-recipe/meta.yaml

```
@@ -13,36 +13,53 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-
-requirements:
-  build:
-    - python >=3.6
-    - pip
-    - setuptools_scm
-  run:
-    - python >=3.6
-    - coloredlogs
-    - happi >=1.6.0
-    - numpy
-    - ophyd
-    - prettytable
-    - pydm
-    - pyqt >=5
-    - qtawesome
-    - qtpy
-    - typhos >=1.0.0
 
 test:
   imports:
     - lightpath
   requires:
     - ipython
-    - matplotlib
     - pytest <7.2.0
     - pytest-qt
 
+outputs:
+  - name: lightpath-base
+    build:
+      noarch: python
+      script: python -m pip install . -vv
```

```suggestion
      script: {{ PYTHON }} -m pip install . -vv
```
I'll look for the docs on this

---

https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html

---

https://github.com/pcdshub/lightpath/actions/runs/10564732964/job/29267769359#step:12:49
>Attempting to finalize metadata for lightpath-base
Error: Failed to render jinja template in /home/runner/work/lightpath/lightpath/conda-recipe/meta.yaml:
'PYTHON' is undefined

---

https://github.com/pcdshub/pcdsdevices/blob/b33b7c4d6a372cb877458f7e446c90d0d083518f/conda-recipe/meta.yaml#L15

---

Maybe this was the warning shot that suggests that something deeper was wrong with the conda build process

---

I see now that the recipes that do the *-base thing don't use `{{ PYTHON }}`, my bad for the red herring